dts/xtensa/espressif/esp32.dtsi

```diff
@@ -354,6 +354,17 @@
 			interrupt-parent = <&intc>;
 			status = "disabled";
 		};
+
+		dac: dac@3ff48800 {
+			compatible = "espressif,esp32-dac";
+			reg = <0x3ff48800 0x100>;
+			interrupts = <RTC_CORE_INTR_SOURCE>;
```

Comment:
Is this the correct interrupt for DAC ?

Reply:
SENS peripheral is using RTC interrupt source.